LEGO1/legoroi.cpp

```diff
@@ -1,16 +1,90 @@
 #include "legoroi.h"
 
+#include <string.h>
+
+// SIZE 0x14
+typedef struct {
+	const char* name;
+	MxS32 red;
+	MxS32 green;
+	MxS32 blue;
+	MxS32 unk10;
+} roi_color_alias;
+
+// 0x100dbe28
+double g_unk_roiConstant = 0.00392156862745098;
```

Comment:
This seems to be `1/255`. I'm assuming it's used to normalize the RGB color values to a value in the range 0-1